targets/targets.json

```diff
@@ -15,6 +15,7 @@
         "default_lib": "std",
         "bootloader_supported": false,
         "static_memory_defines": true,
+        "ulib_supported":false,
```

Comment:
The name of the library is microlib so I'd rather that you use a more descriptive parameter like `microlib_supported`